src/components/ActionComplete.vue

```diff
@@ -135,13 +135,15 @@ export default {
       },
       createCampaignLetter(sessionId) {
         // Creates campaign letter with lob api.
+        let userCustomization = this.$store.state.userCustomization
```

Comment:
It is more idiomatic to use a getter in the Vuex store or a computed property to access state. The rest of the application uses the computed property route. Check these links out:
https://vuex.vuejs.org/guide/getters.html
https://vuex.vuejs.org/guide/getters.html#property-style-access

Reply:
The userCustomization is being sent now 😎

Reply:
![image](https://user-images.githubusercontent.com/36219292/208221072-3032d211-d46e-4967-8bc2-0aab210fbd03.png)